src/client/clientmap.h

```diff
@@ -196,7 +197,7 @@ class ClientMap : public Map, public scene::ISceneNode
 	bool m_needs_update_transparent_meshes = true;
 
 	std::map<v3s16, MapBlock*, MapBlockComparer> m_drawlist;
-	std::map<v3s16, MapBlock*> m_drawlist_shadow;
+	std::unordered_map<v3s16, MapBlock*> m_drawlist_shadow;
```

Comment:
changing the draw order looks dangerous, are you sure this is correct?
@x2048

Reply:
it's a good question, i didn't noticed anything really visible on my side.
the drawlist is written on the shadow renderer to make shadow rendered from player pos to long radius, then i think it's not important there, why do you want to render shadows from block pos in a single direction order ?

Reply:
ok just checked, there is a code part requiring ordering on clientMap, let's revert that